src/util/PromiseUtil.ts

```diff
@@ -24,10 +24,8 @@ export abstract class PromiseUtil {
     }
     return p.then(() => result);
   }
-  static sleepPromise<H>(ms, returnVal?: H): Promise<H> {
-    return new Promise<H>((resolve) => {
-      setTimeout(() => { resolve(returnVal); }, ms);
-    });
+  static async sleepPromise(ms): Promise<any> {
```

Comment:
better keep the generic type here 

`static sleepPromise<H>(ms): Promise<H> {`

Reply:
why? type is not used.